x509/cert_pool.go

```diff
@@ -128,3 +142,10 @@ func (s *CertPool) Subjects() (res [][]byte) {
 	}
 	return
 }
+
+// Certificates returns a list of parsed certificates in the pool.
+func (s *CertPool) Certificates() []*Certificate {
+	out := make([]*Certificate, 0, len(s.certs))
```

Comment:
Because you can still break up the underlying slice in the CertPool since slices are quasi-by-reference.